src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
+    private static final int RIGHT_TRIANGLE_NUMBER = 2;
+    private int sid;
+    private String color;
+    public RightTriangle() {
+        this.sid = sid;
+        this.color = color;
```

Comment:
The constructor 'RightTriangle()' should not be empty and should take parameters to initialize the 'sid' and 'color' fields. Also, the current implementation is assigning the fields to themselves rather than to the parameters provided to the constructor.